webcompat/form.py

```diff
@@ -103,6 +103,7 @@ def get_form(form_data):
     ua_header = form_data['user_agent']
     # Populate the form
     bug_form.browser.data = get_browser(ua_header)
+    bug_form.details.data = json.dumps(form_data.get('details'))
```

Comment:
Yeah you ran into the same type of issues I didn't want to go into in 
https://github.com/webcompat/webcompat.com/pull/2582#discussion_r218993713
This will kind of work but leads to unintended consequences one example is:

![capture d ecran 2018-10-18 a 10 25 50](https://user-images.githubusercontent.com/505230/47125887-c5453100-d2c0-11e8-97fb-74f2706c3320.png)

`@font-face` is an active link.

unintended Notifications hell for people.

I guess this can be fixed before doing:
https://github.com/webcompat/webcompat.com/pull/2658/files#diff-f1975977b06efb529b8c60f23a7bc4eeL320

because if we do in only on the client side on webcompat.com, we don't solve the notifications issue. :/

I have the feeling there is a missing type in the webplatform to be able to store JSON directly into form values.

I wonder what would do a 

```python
console_textarea = TextAreaField("console_log", default=console_log_dumps)
``` 

then 

```css
textarea.console {visibility: hidden}
```

Reply:
I like this idea!

Tom showed me this JSON formattiner thing: https://azimi.me/json-formatter-js/

We could toss all of the details into a textarea (to avoid weird linkage), and render it fancy-like on the client? It's perhaps overkill, but worth at least attempting in a branch. 

Maybe in the meantime, we just toss the console logs in backticks in this PR, and figure out the nicer solution in #2659 ?

```
no emails for @fontface
```